src/main/java/team05/integrated_feed_backend/common/BaseEntity.java

```diff
@@ -0,0 +1,29 @@
+package team05.integrated_feed_backend.common;
+import jakarta.persistence.Column;
+import jakarta.persistence.MappedSuperclass;
+import jakarta.persistence.PrePersist;
+import jakarta.persistence.PreUpdate;
+import lombok.Getter;
+
+import java.time.LocalDateTime;
+
+@Getter
+@MappedSuperclass
+public abstract class BaseEntity {
+    @Column(nullable = false, updatable = false)
+    private LocalDateTime createdAt;
+
+    @Column(nullable = false)
+    private LocalDateTime updatedAt;
+
+    @PrePersist
+    protected void onCreate() {
+        LocalDateTime now = LocalDateTime.now();
+        createdAt = now;
+        updatedAt = now;
+    }
+    @PreUpdate
+    protected void onUpdate() {
+        updatedAt = LocalDateTime.now();
+    }
+}
```

Comment:
💭 혹시 SpringDataJpa에서 제공하는 Auditing 기능을 사용하지 않으신 이유가 있으신가요?
```suggestion
@Getter
@MappedSuperclass
@EntityListeners(AuditingEntityListener.class)
public abstract class BaseEntity {
    @Column(nullable = false, updatable = false)
    @CreatedDate
    private LocalDateTime createdAt;

    @Column(nullable = false)
    @LastModifiedDate
    private LocalDateTime updatedAt;
}
```
요렇게 쓰고, jpaConfig 파일 만들어서 아래 내용만 추가하면 되거든요!
```java
@Configuration
@EnableJpaAuditing
public class JpaConfig {

}
```
차이가 크게 없긴한데, 조금 더 자동화된(?) 느낌인 것 같아서 추천합니당😉

그리고 날짜의 경우에는 Swagger로 내려갈 때, 객체 형식으로 내려갈 수 있어서 필드 위에
`@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'")` 요 애너테이션을 미리 추가해놔도 좋을 것 같아요!

Reply:
수정했습니다 감사해요!!! :green_heart:  code: 8103000